src/_includes/layouts/base.njk

```diff
@@ -5,6 +5,27 @@
     <title>{% if '/' === page.url %}V8 JavaScript engine{% else %}{{ (renderData.title or title) | markdown | striptags | decodeHtmlEntities }} · V8{% endif %}</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="color-scheme" content="dark light">
+    <script>
+      // If `prefers-color-scheme` is not supported, fall back to light mode.
+      // In this case, light.css will be downloaded with `highest` priority.
+      if (window.matchMedia('(prefers-color-scheme: dark)').media === 'not all') {
```

Comment:
Let's not do this. This only affects old IE, which we don't actively support anyway, and the site works fine in dark mode.

Reply:
i.e. can we just remove this entire `<script>`?

Reply:
> and the site works fine in dark mode

I actually wonder what will happen to styles below if `prefers-color-scheme` is not supported. Are you saying the dark one will be loaded? I would think neither one will, but I don't know much about how unsupported media selectors work.

Reply:
They get loaded, but [both with low priority](https://blog.tomayac.com/2018/11/08/why-browsers-download-stylesheets-with-non-matching-media-queries-180513/). 

All up for removing, but one other browser we *might* want to consider is KaiOS.

Reply:
Loaded, yes, but neither one is applied IIUC?

Reply:
Correct. This is why the inline script injects one of them (`light.css`) manually, so it gets loaded with highest priority and is applied, since `main.css` relies on the variable definition made therein. Try my blog on a KaiOS device to see it in action (or a [VM](https://www.browserling.com/browse/win/7/firefox/48/https%3A%2F%2Fblog.tomayac.com%2F)).